types/next/router.d.ts

```diff
@@ -14,17 +14,27 @@ export interface SingletonRouter {
     ready(cb: RouterCallback): void;
 
     // router properties
-    readonly components: { [key: string]: { Component: React.ComponentType<any>, err: any } };
+    readonly components: {
+        [key: string]: { Component: React.ComponentType<any>; err: any };
```

Comment:
You have a semicolon after `<any>;` and I think you want a comma.

Reply:
That is the behaviour of prettier and it is correct, you can read more here https://github.com/prettier/prettier/issues/1944